vnc_auto.html

```diff
@@ -198,9 +222,10 @@
                            'local_cursor': WebUtil.getQueryVar('cursor', true),
                            'shared':       WebUtil.getQueryVar('shared', true),
                            'view_only':    WebUtil.getQueryVar('view_only', false),
-                           'onUpdateState':  updateState,
+                           'updateState':  updateState,
```

Comment:
This should still be onUpdateState (otherwise the status bar doesn't function correctly)